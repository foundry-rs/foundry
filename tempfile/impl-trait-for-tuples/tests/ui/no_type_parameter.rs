#[impl_trait_for_tuples::impl_for_tuples(1)]
trait Test {
	type Test;
}

fn main() {}
