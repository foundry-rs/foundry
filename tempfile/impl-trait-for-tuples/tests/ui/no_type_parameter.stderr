error: Not supported by full-automatic tuple implementation. Use semi-automatic tuple implementation for more control of the implementation.
 --> tests/ui/no_type_parameter.rs:3:2
  |
3 |     type Test;
  |     ^^^^
